### PR TITLE
Fix encrypted output to be unbounded & Serde camel case

### PR DESCRIPTION
**Pull request by AhmedKorim**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?
- `ExtData`  encrypted output fields are bounded however for protocol substrate node now they are unbounded
- `ExtData` , `ProofData` missing the serde  `#[serde(rename_all = "camelCase")]`, Thus the webbsocket payload isn't in camel case
Issue Number: N/A

## What is the new behavior?
- Fix the `ExtData`
- Camel casing the WebSocket payload keys for transaction relaying

## Does this PR introduce a breaking change?
- [X] Yes
- [ ] No

## Other information
`ExtData.encrypted_output1` ,and `ExtData.encrypted_output2` follow the smae type as the Proof `Bytes` for evm ,and `Vec<u8>` for substrate

### CODE REVIEW

File: config/development/local-substrate/substrate.toml [hunk 1]
```diff
@@ -2,8 +2,10 @@ port = 9955
...
-http-endpoint = "http://localhost:9933/"
-ws-endpoint = "ws://localhost:9944/"
+http-endpoint = "http://localhost:9944"
+ws-endpoint = "ws://localhost:9944"
 runtime = "WebbProtocol"
 suri = "//Alice//stash"
-pallets = []
+[[substrate.localnode.pallets]]
+pallet = "VAnchorBn254"
+events-watcher = { enabled = true ,polling-interval = 10000 , print-progress-interval = 0}
```
Comment (shekohex) [inline config/development/local-substrate/substrate.toml:11]:
```suggestion
[[substrate.localnode.pallets]]
pallet = "VAnchorBn254"
events-watcher = { enabled = true, polling-interval = 10000, print-progress-interval = 0 }
```

File: config/development/local-substrate/substrate.toml [hunk 2]
```diff
@@ -2,8 +2,10 @@ port = 9955
 
 [substrate.localnode]
 enabled=true
-http-endpoint = "http://localhost:9933/"
-ws-endpoint = "ws://localhost:9944/"
+http-endpoint = "http://localhost:9944"
+ws-endpoint = "ws://localhost:9944"
```
Comment (dutterbutter) [inline config/development/local-substrate/substrate.toml:6]: These shouldn't be the same port, no?

Comment (AhmedKorim) [reply config/development/local-substrate/substrate.toml:6]: Not sure if  https is been used for substrate but it works with this configuration, @shekohex  please confirm

Comment (dutterbutter) [reply config/development/local-substrate/substrate.toml:6]: The default substrate http port is `9933`

**Review by shekohex — APPROVED** [review]
LGTM!

**Pull request merged**